.gitignore

```diff
@@ -1 +1,3 @@
 .DS_Store
+Gemfile.lock
```

Comment:
For libraries, you can't guarantee many details about the environment they run in, particularly what version of a dependency is being used.

Ignoring the lockfile forces your tests to always run against the latest version of your stated dependencies, theoretically giving you an early warning that a new version has broken something.